front/internals/scripts/extract-intl.js

```diff
@@ -163,4 +163,9 @@ const extractFromFile = async (fileName) => {
   }
 
   process.exit();
-}());
+}
+
+(async function main() {
+  extractMessages('app/**/messages.*s', 'app/**/{a,A}dmin/**/messages.*s', 'app/translations/');
+  extractMessages('app/**/{a,A}dmin/**/messages.*s', '', 'app/translations/admin/');
+}());
```

Comment:
Wrap the main execution in a function (called `extractMessages`), and run it twice, once for non-admin stuff, once for admin stuff.